src/group_theory/order_of_element.lean

```diff
@@ -771,6 +771,16 @@ begin
   exact pow_card_eq_one,
 end
 
+@[to_additive nsmul_eq_nsmul_mod_card] lemma pow_eq_pow_mod_card (n : ℕ) :
```

Comment:
```suggestion
@[to_additive nsmul_eq_mod_card] lemma pow_eq_mod_card (n : ℕ) :
```
Would be more consistent with the name of the existing lemma `pow_eq_mod_order_of`. Can you either change the new names to match the old ones or vice versa?